.github/workflows/codestyle.yml

```diff
@@ -2,9 +2,6 @@ name: codestyle
 
 on:
   push:
-    branches:
-    - dev
```

Comment:
Why remove codestyle check on pushes into dev?

Reply:
In that case, codestyle will trigger twice on pushes into branches with open PRs into dev.

Reply:
What branches do you want it to run codestyle on? Any branch that has a PR into dev will be checked. Other branches don't really need codestyle (and if they do, then it can be triggered).

Reply:
So we leave codestyle as it was before? (only on branches that have a PR into dev)